services/web/client/source/class/qxapp/data/model/Node.js

```diff
@@ -760,8 +760,6 @@ qx.Class.define("qxapp.data.model.Node", {
         // FIXME: this is temporary until the reverse proxy works for these services
         if (this.getKey().includes("neuroman") || this.getKey().includes("modeler")) {
           srvUrl = "http://" + window.location.hostname + ":" + publishedPort + srvUrl;
-        } else if (this.getKey().includes("3d-viewer")) {
-          srvUrl = "http://" + window.location.hostname + ":" + publishedPort + entryPoint;
         }
```

Comment:
one hack less 👍

Reply:
will then only work from version 2.11.0 of paraview, I did not try other versions.